lua/starfall/libs_sv/prop.lua

```diff
@@ -481,9 +481,20 @@ function props_library.createSent(pos, ang, class, frozen, data)
 				
 				checkluatype(value, org[2], nil, "Parameter: " .. param)
 				
-				enttbl[org[1]] = value
-			else
+				if istable(value) then
```

Comment:
Not all tables are a color, e2 included for example. This also wont allow for tables structured with rgb indexes, this is thje reason castColor exists.